web/static/js/settings/connection-feature-toggle.js

```
@@ -86,5 +86,58 @@
+    if (!connID) {
+      console.error('swVerifyPathAfterUpdateAfterRequest: missing data-conn-id on toggle; cannot sync state', triggerEl);
+      return;
+    }
+    var errEl = document.getElementById('verify-path-error-' + connID);
+    if (event.detail.successful) {
+      if (errEl) {
+        errEl.textContent = '';
+        errEl.classList.add('hidden');
+      }
+      try {
+        var resp = JSON.parse(event.detail.xhr.responseText || '{}');
```

Biome (2.5.0)

[error] This var should be declared at the root of the enclosing function.

(lint/correctness/noInnerDeclarations)